apps/web/src/server/applications/case/documentation-metadata/documentation-metadata.validators.js

```diff
@@ -28,8 +28,11 @@ export const validatorsDispatcher = async (request, response, next) => {
 		'published-date': validateDocumentationMetaDatePublished
 	};
 
-	if (validators[metaDataName]) {
-		return validators[metaDataName](request, response, next);
+	const validatorKey = Object.entries(validators).find((entry) => entry[0] === metaDataName);
```

Comment:
This also looks a bit obscure, would the following suggested change pass the scan?:
```
	const validator = validators[metaDataName];
	if (typeof validator === 'function') {
		return validator(request, response, next);
	}
```